src/components/subPages/formInput/formInput.tsx

```diff
@@ -0,0 +1,42 @@
+import {TextField, TextFieldProps} from '@mui/material'
+import React, {FC} from 'react'
+import {Controller, useFormContext, get} from 'react-hook-form'
+
+type IFormInputProps = {
+	name: string
+} & TextFieldProps
+
+const style = {

```

Comment:
Style inline są mocno nie wskazane. Teraz jakbyś chciał nadpisać klasą width lub marginTop to miałbyś z tym problem, bo style inline są ważniejsze niż selectory z css

Reply:
Lepiej dodać klasę konretnie dla FormInput